addons/mail/static/src/models/chat_window/chat_window.js

```diff
@@ -8,6 +8,33 @@ function factory(dependencies) {
 
     class ChatWindow extends dependencies['mail.model'] {
 
+        /**
+         * @override
+         */
+        static create(data) {
+            const chatWindow = super.create(data);
+            chatWindow.env.messagingBus.on(
+                'hide_home_menu',
+                chatWindow,
+                chatWindow._onHideHomeMenu.bind(chatWindow)
+            );
+            chatWindow.env.messagingBus.on(
+                'show_home_menu',
+                chatWindow,
+                chatWindow._onShowHomeMenu.bind(chatWindow)
+            );
+            return chatWindow;
+        }
+
+        /**
+         * @override
+         */
+        delete() {
+            this.env.messagingBus.off('hide_home_menu', this, this._onHideHomeMenu.bind(this));
+            this.env.messagingBus.off('show_home_menu', this, this._onShowHomeMenu.bind(this));
```

Comment:
Just bind `_on<Show/Hide>HomeMenu` before `.on()` and then just pass `chatWindow._on<Show/Hide>HomeMenu` as function to `.on()` and `.off()`, without binding them again (since they are already bound).